optuna/samplers/_partial_fixed.py

```diff
@@ -92,16 +92,17 @@ def sample_independent(
     ) -> Any:
 
         # If param_name isn't in self._fixed_params.keys(), param_value is set to None.
-        param_value = self._fixed_params.get(param_name)
-
-        if param_value is None:
+        # And if param_value is `None` itself it should be `None`
```

Comment:
I think we can remove this comment because this if-statement is not hard to read.

Reply:
Hmm, my comment is not rendering as I expected. I mean, the two lines (`# If param_name isn't ...` and `# And if param_value is ...`) can be removed.